whois-commons/src/main/java/net/ripe/db/whois/common/override/OverrideCredentialValidator.java

```diff
@@ -23,11 +23,26 @@ public OverrideCredentialValidator(final IpRanges ipRanges,
         this.userDao = userDao;
     }
 
+    public boolean isAllowedAndValid(final boolean isTrusted, final UserSession userSession,
+                                     final OverrideCredential overrideCredential,
+                                     final ObjectType objectType){
+        if (overrideCredential == null) {
+            return false;
+        }
+        return overrideCredential.getOverrideValues().filter(values -> (isTrusted || isAllowedBySSO(userSession, values.getUsername())) &&
```

Comment:
Can you return early if it is not trusted? will be much more readable than a check in streams

Reply:
mmmm....I cannot do that if it is a list
I can just get the first value...in GET request you only can have one override